dojo/tools/semgrep/parser.py

```diff
@@ -0,0 +1,49 @@
+import json
+from dojo.models import Finding
+from dojo.tools.semgrep.models import SemgrepJSONResult
+
+
+class SemgrepJSONParser(object):
+
+    def __init__(self, filehandle, test):
+        tree = self.parse_json(filehandle)
+
+        self.items = []
+        if tree:
+            results = tree.get('results')
+
+            if not results:
+                return
+
+            for item in results:
+                title = item['check_id']
+                f = SemgrepJSONResult(item['extra'], item['path'], item['start'], item['end'])
```

Comment:
could you use a more descriptive variable name, i.e. semgrep_result instead of f?